helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -18,6 +18,15 @@ receivers:
   k8s_cluster:
     auth_type: serviceAccount
     metadata_exporters: [signalfx]
+  {{- if .Values.otelK8sClusterReceiver.k8sEventsEnabled }}
+  smartagent/kubernetes-events:
+    type: kubernetes-events
+    whitelistedEvents:
+    - reason: Created
+      involvedObjectKind: Pod
+    - reason: SuccessfulCreate
+      involvedObjectKind: ReplicaSet
```

Comment:
Let's set this list to the following defaults
```
      - reason: Created
        involvedObjectKind: Pod
      - reason: Unhealthy
        involvedObjectKind: Pod
      - reason: Failed
        involvedObjectKind: Pod
      - reason: FailedCreate
        involvedObjectKind: Job
```

Reply:
added default.